### DISCUSSION TITLE
INMP441 and microphone inputs in general

### DISCUSSION THREAD

**Discussion by mmcc-xx** [pschatzmann/arduino-audio-tools#816] [Q&A] [answered]
I'm successfully running https://github.com/pschatzmann/arduino-audio-tools/tree/main/examples/sandbox/streams-i2s-webserver_aac

My use case is capturing audio for use with BirdNET-PI for bird identification. Seems to be working just fine.

A couple of questions though...
If I try to increase either the sample rate or the bits per sample from the values used in the example, I just get static. Is this a limitation of the mic or the board or something else entirely?

Is there a board that I could connect an external dynamic mic to that outputs I2S?

Thanks,
Clark

**Comment by pschatzmann** [comment, marked as answer]
I would not recommend to use any aac or mp3: the overhead on the microcontroller is just to big. Therefore I suggest to stick with the WAV format, which is very efficient to process.

There are also many other more efficient codecs: you can't use them in your browser however and you would need to to use them together with something like FFmpeg on the desktop.

I suggest to have a look at the LyraT audio board: it has some regular audio in and out connectors and some built in microphones.
For your purpose I can not recommend the simillar AI Thinker AudioKit board if you plan to provide some external audio input.

**Reply by mmcc-xx** [reply]
Thanks! ffmpeg is indeed the receiver of the stream, and I don't really care about listening in. What codec would you recommend?

I found the INMP441 microphone to be not terribly sensitive. Have you tried out the MAX9814 microphones and are they any better at picking up fainter sounds?

I'll take a look at the LyraT board. Thanks!

**Comment by pschatzmann** [comment]
I suggest any codec which is used by bluetooth e.g. SBC
